app/controllers/home_controller.rb

```diff
@@ -3,10 +3,10 @@ class HomeController < ApplicationController
   before_action :update_last_viewed_at
 
   def show
-    @q = current_organization.transactions.ransack(params[:q])
+    @q = current_organization.transactions.ordered.ransack(params[:q])
```

Comment:
@yunixon for ransack you should set default ordering another way:
```rb
@q = ...
@q.sorts = 'date asc' if @q.sorts.blank?
@transactions = ...
```
With this default ordered column will be shown in UI and allow user to set another order.